mixer/adapter/signalfx/config/config.proto

```diff
@@ -0,0 +1,78 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// $title: SignalFx
+// $description: Adapter that sends Istio metrics to SignalFx
+// $location: https://istio.io/docs/reference/config/policy-and-telemetry/adapters/signalfx.html
+// $front_matter: aliases:
```

Comment:
Since this is a new adapter, you don't need the redirect to be generated, so just remove this line and the following.